AdyenComponents/PayTo/PayToItemsProvider.swift

```diff
@@ -44,20 +44,41 @@ internal class PayToItemsProvider: PayToItemsProviding {
     }
     
     private enum ValidationRegex {
-        static let phone = #"^\+[0-9]{1,3}-[1-9]{1,1}[0-9]{1,29}$"#
+        static let phone = #"^[1-9]{1,1}[0-9]{1,29}$"#
         static let abn = #"^((\d{9})|(\d{11}))$"#
         static let organizationId = #"^[!-@\[-~][ -@\[-~]{0,254}[!-@\[-~]$"#
-        static let bsb = #"^\d{6}"#
-        static let accountNumber = #"^\[ -~]{1,28}$"#
+        static let bsb = #"^\d{6}$"#
+        static let accountNumber = #"^[ -~]{1,28}$"#
```

Comment:
We don't receive this from backend, right?

Reply:
Unfortunately not

Reply:
Can we challenge this and suggest fetching these rules from server?

Reply:
I asked at the very beginning and it is not possible. Also they deem these to be highly unlikely to change which adds another reason not to spend effort on such solutions.